policy/modules/roles/sysadm.te

```diff
@@ -12,6 +12,14 @@ policy_module(sysadm, 2.15.3)
 ## </desc>
 gen_tunable(allow_ptrace, false)
 
+## <desc>
+## <p>
+## Allow sysadm to read/write to fifo files inherited from
+## a domain allowed to change role.
+## </p>
+## </desc>
+gen_tunable(sysadm_allow_rw_inherited_fifo, true)
```

Comment:
Since this allows the sysadm to possibly be influenced by unpriv processes, this should default to `false`.